modules/nickserv/multimark.c

```diff
@@ -1032,6 +1046,22 @@ ns_cmd_multimark(struct sourceinfo *si, int parc, char *parv[])
 				break;
 		}
 
+		MOWGLI_ITER_FOREACH_SAFE(n, tn, rl->head)
+		{
+	                rm = n->data;
```

Comment:
What happened with the indentation in there?

Reply:
oops, thanks!